arcgis-ios-sdk-samples/Info.plist

```diff
@@ -4,6 +4,8 @@
 <dict>
 	<key>CFBundleDevelopmentRegion</key>
 	<string>en</string>
+	<key>CFBundleDisplayName</key>
+	<string>Samples</string>
```

Comment:
The current bundle display name (on the App Store) is "iOS Samples". I'm fine with using just "Samples", although it  rather generic. @dg0yal Do you have a preference?

Reply:
I am fine with Samples